packages/react-composites/src/composites/MeetingComposite/styles/MeetingCompositeStyles.ts

```diff
@@ -18,3 +18,24 @@ export const compositeOuterContainerStyles: IStackStyles = {
     flexWrap: 'wrap'
   }
 };
+
+/**
+ * @private
+ */
+export const NotificationIconStyles: IIconStyles = {
+  root: {
+    color: '#0078D4'
```

Comment:
This should likely come from `theme`. Try dark theme in storybook and see what the notification icon looks like / should look like.

Reply:
yeah for this you will need to use `useTheme` and use `useTheme().palette.themePrimary`.
You must use useTheme inside the component render function however.